kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/PartitionDeserializer.java

```diff
@@ -0,0 +1,26 @@
+package org.axonframework.extensions.kafka.eventhandling;
```

Comment:
Since this class is specific to the `KafkaTrackingToken`, I would expect it to reside in the `org.axonframework.extensions.kafka.eventhandling.consumer.streamable` package.
So, please move this file to the right directory.